### ISSUE TITLE
Type.parameter.rust semantic token not detected on standalone rust files.

### ISSUE THREAD

**Issue by Rahtgaz** [rust-lang/rust-analyzer#22230]
rust-analyzer 2026-04-27
rustc 1.95.0 (59807616e 2026-04-14)
nvim

```rust
fn main() {
    foo(12, 23);
}

fn foo(bar: i32, baz: i32) -> i32 {
    bar + baz
} 
```

The `:Inspect` command on `baz` or `baz` variables inside the `foo()` function body, will only return `@lsp.type.parameter.rust` when a Cargo.toml file is detected.

<img width="819" height="463" alt="Image" src="https://github.com/user-attachments/assets/66aca4ae-cbb0-4a8e-b0f9-f574ebf37ea7" />
<img width="790" height="465" alt="Image" src="https://github.com/user-attachments/assets/31154153-fea9-4934-8ea5-2236871dac27" />

**Comment by ChayimFriedman2** [comment]
This is a token from semantic highlighting, and like everything in rust-analyzer it cannot operate without an active Rust project.
